app/src/test/java/com/pelukron/ListaTest.java

```diff
@@ -0,0 +1,30 @@
+package com.pelukron;
+
+import jdk.jfr.Description;
```

Comment:
Are you using this library?

Reply:
Not really, i am not sure where i am imported but anyway, i was looking this method [DisplayName](https://www.digitalocean.com/community/tutorials/junit-display-name-displayname)

Reply:
> Not really, i am not sure where i am imported but anyway, i was looking this method [DisplayName](https://www.digitalocean.com/community/tutorials/junit-display-name-displayname)

As summary,  i need to remove this unused dep